conf/application.conf

```diff
@@ -146,6 +146,8 @@ microservice {
     feedback.url = "http://localhost:9514/feedback/ATS/personal"
     login.url = "http://localhost:9025/gg/sign-in?continue=http://localhost:9217/annual-tax-summary"
     login-callback.url = "http://localhost:9217/annual-tax-summary"
+    paye-tax-year=2019
+    sa-auth-only=false
```

Comment:
Should year be specified in config or calculated?

`sa-auth-only` flag should probably be set to true in `app-config-base` as a precaution if set to false here.

Reply:
Will probably remove this all together now we plan to have two endpoints.